lib/std/crypto/ml_kem.zig

```diff
@@ -174,7 +172,9 @@ pub const kyber_d00 = struct {
     });
 };
 
-pub const ml_kem_01 = struct {
+pub const ml_kem_01 = @compileError("deprecated: final version of the specification has been published, use ml_kem instead");
+
+pub const ml_kem = struct {
     pub const MLKem512 = Kyber(.{
```

Comment:
[Avoid Redundant Names in Fully-Qualified Namespaces](https://ziglang.org/documentation/master/#Avoid-Redundant-Names-in-Fully-Qualified-Namespaces)

Reply:
Renamed to `nist` and the draft version to `d00` for consistency.